packages/aragon-client/src/index.js

```diff
@@ -19,12 +21,29 @@ export const AppProxyHandler = {
   }
 }
 
+export class NetworkAPI extends EventEmitter {
+  constructor (rpc) {
+    super()
+    this.rpc = rpc
+    this.observable = new ReplaySubject(1)
+    this.rpc.sendAndObserveResponses('network')
+      .pluck('result')
+      .do(value => { this.emit('update', value) })
+      .subscribe(this.observable)
+  }
+
+  get () {
+    return this.observable.toPromise()
```

Comment:
Not 100% sure what the behaviour is with a `ReplaySubject()`, but I think we still need to do a `.first()` to end the observable before calling `toPromise()`?

Reply:
Naw, just that `this.observable.toPromise()`'s promise won't resolve until `this.observable` itself ends, so we need to use `.first()` to end it.